[Style guide] Grammar and Formatting (#9263)
cloudflare/cloudflare-docs: content/style-guide/grammar-and-formatting/file-types-and-extensions.md

---
pcx_content_type: Concept
title: File types and extensions
---

# File types and extensions

You can refer both to file types and file extensions in documentation:

+ File types: Accepted file formats are JSON and YAML.
+ File extensions: Upload the `.txt` file to the dashboard.

> 
> When referring to a specific file type: always use regular text formatting, write the file type in uppercase, and avoid the initial dot (you are referring to the file type, not the file extension).

Reviewer: ```suggestion

When referring to a specific file type, always use regular text formatting, write the file type in uppercase, and avoid the initial dot (you are referring to the file type, not the file extension).
```

---
pcx_content_type: Concept
title: File Types and Extensions
---

## File Types and Extensions

You can refer both to file types and file extensions in documentation:

> Referring to file types: Accepted file formats are JSON and YAML. 
> Referring to file extensions: Upload the `.txt` file to the dashboard.

Reviewer: ```suggestion
- Referring to file types: Accepted file formats are JSON and YAML. 
- Referring to file extensions: Upload the `.txt` file to the dashboard.
```

> + Referring to file types: Accepted file formats are JSON and YAML. 
> + Referring to file extensions: Upload the `.txt` file to the dashboard.

Reviewer: ```suggestion
+ File types: Accepted file formats are JSON and YAML. 
+ File extensions: Upload the `.txt` file to the dashboard.
```

When referring to a specific file type, always:

+ use regular text formatting
+ write the file type in uppercase
+ avoid the initial dot (you are referring to the file type, not the file extension)

> 

Reviewer: it might be better if we just make this into one sentence rather than bullet points. Same goes for the other sections below this.

When referring to a specific extension, always:

+ monospace the file extension
+ include a dot
+ write the extension in lowercase

> + always specify a noun after mentioning a file extension, e.g. "file" or "file extension". Don't leave the extension hanging

Reviewer: ```suggestion
+ always specify a noun after mentioning a file extension, e.g. "file" or "file extension". Do not leave the extension hanging
```

    + ✅Download the `.txt` file.
    + ❌Download the `.txt`.

Other examples:

> The accepted file formats are YAML (files with a `.yml` or `.yaml` file extension) and JSON (files with a `.json` file extension).
> Additionally, this plist can be wrapped in a `.mobileconfig` file.
> If you'd like to test how this feature works, here is a sample `.csv` file.
> Concatenate the string to a text-based `.pem` file.

Reviewer: ```suggestion
- The accepted file formats are YAML (files with a `.yml` or `.yaml` file extension) and JSON (files with a `.json` file extension).
- Additionally, this plist can be wrapped in a `.mobileconfig` file.
- If you'd like to test how this feature works, here is a sample `.csv` file.
- Concatenate the string to a text-based `.pem` file.
```
